composer.lock

```diff
@@ -4743,9 +4743,6 @@
             ],
             "description": "PrestaShop module productcomments",
             "homepage": "https://github.com/PrestaShop/productcomments",
-            "support": {
```

Comment:
Which version of composer did you use? These elements shouldn't have to be removed :thinking:

Reply:
Composer version 1.10.15 2020-10-13 15:59:09

Reply:
You can still use it to install modules, but when updating this file, we must use the latest composer version, otherwise, we will remove and add many parts of it each time we update our modules, which is not consistent